src/cli/cli.cpp

```diff
@@ -4212,6 +4255,16 @@ template <> otError Interpreter::Process<Cmd("pskcref")>(Arg aArgs[])
     {
         otPskcRef pskcRef;
 
+        /**
+         * @cli pskcref (set)
+         * @code
+         * pskc 0x20017
+         * Done
+         * @endcode
+         * @cparam pskc @ca{keyref}
+         * @par api_copy
+         * #otThreadSetPskcRef
```

Comment:
Can you update the comment for this API to say this for the first line:

"Set the Key Reference to the Thread PSKc."

And also remove the "to this function" in the later paragraph so it reads:

"Upon success, this will also invalidate..."

API comments are in this file: https://github.com/openthread/openthread/blob/c10f650468fd5984b474de707fc2eae5f9234fdc/include/openthread/thread_ftd.h#L692

Just update there, and re-run the import to verify it comes through.  You'll need to increment the API version too as part of this (required any time the API headers are changed):

https://github.com/openthread/openthread/blob/c10f650468fd5984b474de707fc2eae5f9234fdc/include/openthread/instance.h#L56